### PR TITLE
ARC-27 remove TUNNEL_SUBDOMAIN env var from app

### PR DESCRIPTION
**Pull request by rachellerathbone**
The TUNNEL_SUBDOMAIN env var, while intended to be part of ngrok setup, doesn't seem to have any purpose in the app.

I removed it from the following files:

- CONTRIBUTING.md
- tunnel script
- .env.example

And tested the entire flow with both ngrok and localtunnel.

**Ngrok**

Screen Shot 2021-04-28 at 8 51 34 am

Screen Shot 2021-04-28 at 8 53 28 am

I ran through the flow from uploading my app to Jira with `https://rachelles-github-app.ngrok.io/jira/atlassian-connect.json`, added an org and installed, and gave access to the following repo:

Screen Shot 2021-04-28 at 8 54 57 am

I then created a branch named `TEST-13-ngrok-no-tunnel-subdomain`, made a small change, committed it, and pushed. Both the branch and commit are reflected in the corresponding Jira issue.

Screen Shot 2021-04-28 at 8 57 01 am

Screen Shot 2021-04-28 at 8 57 16 am

Screen Shot 2021-04-28 at 8 57 23 am

**Local tunnel**

Screen Shot 2021-04-28 at 8 59 23 am

Screen Shot 2021-04-28 at 9 00 04 am

Same flow as for ngrok (but on branch `TEST-14-lt-no-tunnel-subdomain`) and, once again, can see branch and commit in Jira issue.

Screen Shot 2021-04-28 at 9 04 45 am

Screen Shot 2021-04-28 at 9 05 03 am

Screen Shot 2021-04-28 at 9 05 10 am

In both flows, I also checked that I could see both the installation and the subscription in the db:

Screen Shot 2021-04-28 at 9 01 59 am

Screen Shot 2021-04-28 at 9 03 06 am

### CODE REVIEW

**Review by mboudreau — APPROVED** [review]
Looks good to me :)

File: script/tunnel [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
   echo "TUNNEL_PORT must be set"
   exit 1
-elif [ -z ${TUNNEL_SUBDOMAIN} ]; then
-  echo "TUNNEL_SUBDOMAIN must be set"
-  exit 1
 fi
 
 if is_installed ngrok; then
-  exec ngrok http "${TUNNEL_PORT}" --subdomain="${TUNNEL_SUBDOMAIN}" --log=stdout
+  exec ngrok http "${TUNNEL_PORT}" --log=stdout
```
Comment (saquibkhan) [inline script/tunnel:15]: Q: Can we do a local setup without this? As I understand if we not using the tunnel subdomain a random URL will be generated by ngrok and the same URL needs to be used as DOMAIN in App config and env file. prior to running the script.

**Comment by rachellerathbone** [comment]
See comment thread.

**Review by saquibkhan — APPROVED** [review]

**Comment by rachellerathbone** [comment]
Although approved, closing as we _think_ this might make testing more convoluted.

**Comment by rachellerathbone** [comment]
Although approved, closing as we think this might make testing more convoluted.

**Pull request closed without merging**